src/app/commands/connect/WalletConnect.ts

```diff
@@ -0,0 +1,34 @@
+import { SlashCommand, CommandContext, SlashCreator } from 'slash-create';
+import ServiceUtils from '../../utils/ServiceUtils';
+import discordServerIds from '../../service/constants/discordServerIds';
+import Log from '../../utils/Log';
+
+export default class WalletConnect extends SlashCommand {
+	constructor(creator: SlashCreator) {
+		super(creator, {
+			name: 'connect',
+			description: 'Connect your wallet',
+			guildIDs: [discordServerIds.banklessDAO, discordServerIds.discordBotGarage],
+		});
+	}
+
+	async run(ctx: CommandContext): Promise<any> {
+		// Ignores commands from bots
+		if (ctx.user.bot) return;
+
+		if (ctx.guildID == null) {
+			await ctx.send({ content: 'Please try this command within a discord server.' });
+			return;
+		}
+
+		try {
+			const { guildMember } = await ServiceUtils.getGuildAndMember(ctx.guildID, ctx.user.id);
+			const publicReplyStr = 'Follow this link to verify your address: http://localhost:4200/connect';
```

Comment:
`localhost` isn't quite going to deliver the right end point when this deployed to the backend. Is there an alternative approach to ensure the user arrives at the proper frontend?

Reply:
good catch. 👍 

Have we set up a domain for DEGEN front-end yet? 🤔 💭 

If no, that'll be something I can help with. 🔧